backend/eventd/eventd.go

```diff
@@ -158,6 +158,7 @@ func (e *Eventd) handleMessage(msg interface{}) error {
 		}
 
 		event.Check.MergeWith(prevEvent.Check)
+		event.LastOK = prevEvent.LastOK
```

Comment:
Given that we move LastOK from Event to Check, this will have to change. Also, this isn't the only desired effect we want. If the current event's status is OK, we want to use this event's timestamp as LastOK.

Reply:
This part was just the merge from `prevEvent`. The current event's LastOK is set in `flapping.go` [here](https://github.com/zbintliff/sensu-go/blob/21fb439301d747d270d3f579421b0d040b9b7a25/backend/eventd/flapping.go#L28) if status is OK.

This will be the same if we move `LastOK` into the check as well. Except this line will go into the `MergeWith()`